libs/navigation/bootstrapper.js

```diff
@@ -35,4 +35,39 @@ export default async function bootstrapBlock(miloLibs, blockConfig) {
       loadPrivacy(getConfig, loadScript);
     }, blockConfig.delay);
   }
+
+  /** Jarvis Chat */
+  const isChatInitialized = (client) => !!client?.isAdobeMessagingClientInitialized();
+
+  const redirectToSupport = () => window.location.assign('https://helpx.adobe.com');
+
+  const isChatOpen = (client) =>
+    client?.isAdobeMessagingClientInitialized() && client?.getMessagingExperienceState()?.windowState !== 'hidden';
+
```

Comment:
We can just use above function? 
`const isChatOpen = (client) =>  isChatInitialized && client?.getMessagingExperienceState()?.windowState !== 'hidden';`

Reply:
Done